### ISSUE TITLE
Feedback for the Contact us page (Desktop)

### ISSUE THREAD

**Issue by sashooa** [teamforus/forus-website#9]
### Feedback For [Contact Us](https://github.com/teamforus/WEB-2023-website-forus/issues/158) Page (Desktop):

![Contact Us 1](https://github.com/user-attachments/assets/111df1d9-6eaa-46ad-b2f5-da2e9efadbf1)
1. (1) The distance between image and the header title should be less, (2) the distance between header title and subtitle also must be less, pleas, (3) the distance between subtitle and separation with dashed line and logo of Forus also must be less, please make it as in the design, (4) where are the circles on the background in the header (top) part of the website? (5) Also the banner-image actually is a slideshow and it must have a pagination, please take a look in the design: https://www.figma.com/design/oDL2mEs3bNBNsXJqJAKBJT/Forus.io-v2.0?node-id=1523-9795&t=facVqOXtgOBQqSVj-4
![Contact Us 2](https://github.com/user-attachments/assets/51d26383-dd56-4900-b119-c5e8d8c06fbc)
2. (1) Make less distance between the separator and header title, (2) Less distance between the header title and the contact details, (3) contact details (below label) don't have an indentation, they should be in line with the label, please move contact details to the right, - see how it is in the design: https://www.figma.com/design/oDL2mEs3bNBNsXJqJAKBJT/Forus.io-v2.0?node-id=1523-9868&t=facVqOXtgOBQqSVj-4 , (4) the distance between icon and label-text everywhere is different please make it the same, (5) the social media icons have a wrong size they should be 50 px instead of 40, (6) where is the shadow that should be on hover effect on the social media icon? (7) please make sure that the distance between the map and contact details is correct (!) in the design the distance is much bigger, so please increase the distance, 
![Contact Us 3](https://github.com/user-attachments/assets/bf834c30-50e9-4e91-a04d-5c9dce4ae410)
3. (1) make less distance between contact details and head up of the support section, (2) The grey base on the background of icons with the contacts must be wider - see the design, (3) The section with 4 icons should be in line with the main header and description, (4) icons must the bigger please see the design, (5) blue hover appears not only while the cursor is above the icon but the whole "square" area should be "hot area" with the hover effect, (6) please increase the distance between support and the header of the contact form... (7) please make less distance between header and subtitle, (8) Why are the circles on the background missing? I didn't approve this design "improvement", so please add the circles.😉 - see how it is in the design: https://www.figma.com/design/oDL2mEs3bNBNsXJqJAKBJT/Forus.io-v2.0?node-id=1523-9821&t=facVqOXtgOBQqSVj-4
![Contact Us 4](https://github.com/user-attachments/assets/ac858759-1d3a-40ff-9724-d319e7e04403)
4. (1) please, increase the distance between support and the contact form, (2) please, make less distance between the header title and description of the contact form, (3) the text placeholders in the inputs should be in the same size on each stage (normal, active, error), (4) the button has a wrong size it should be wider, (5) distance between words should be fixed, please make it smaller, (6) Why are the circles on the background missing?😉
![Contact Us 5](https://github.com/user-attachments/assets/9a734dbd-11cf-4318-b201-4c8426a78069)
5. (1) Button must be smaller, please take a look in the design file: https://www.figma.com/design/oDL2mEs3bNBNsXJqJAKBJT/Forus.io-v2.0?node-id=1557-10377&t=facVqOXtgOBQqSVj-4 , (2) Why are the circles on the background missing?😉

**Comment by aghimpu** [comment]
Done in https://github.com/teamforus/forus-frontend-react/tree/feature.website-pages except the banner slideshow part (will need more images to finish it)

**Comment by LindaForus** [comment]
@MichaelForus I agree:
- Mailto should be: info@forus.io

About the telephone number, it it correct that you answer the calls to this number +31 (0) 85 004 39 52 as well @MichaelForus?

**Comment by MichaelForus** [comment]
> About the telephone number, it it correct that you answer the calls to this number +31 (0) 85 004 39 52 as well @MichaelForus?

No, on the old website we used +31 (0) 85 004 39 52, I think this goes to Maarten and often not Support related questions

**Comment by LindaForus** [comment]
@MichaelForus @EmielForus The phonenumber on the new website should indeed be: +31 (0) 85 004 39 52.

This number should link to you, so that the number is reachable. If it is a question for someone else, you can forward it internally or make a callback request.

**Comment by sashooa** [comment]
@MariaForus not yet, I didn’t have enough time for this😉 and before everything, I would like to finish my feedback for the remaining pages, then I will come back to all previous issues 🙏🏻
